scripts/reorder_json_keys.py

```diff
@@ -1,7 +1,7 @@
 import json
 import os
 
-from recipe_scrapers._utils import get_abstract_methods
+from recipe_scrapers._utils import NUTRITION_KEYS, get_abstract_methods
```

Comment:
Instead of exporting a `list` for the nutrition keys and a method to call for the abstract methods, could we make both into methods?

```suggestion
from recipe_scrapers._utils import get_nutrition_keys, get_abstract_methods
```